### PR TITLE
Add support for visio

### PR DESCRIPTION
**Pull request by eboureau**
Add support for visio xml files

### CODE REVIEW

**Comment by eboureau** [comment]
Hi @sindresorhus, I don't see any reason why the node.js 22 check is failing on xo install. 
Any idea?
Thanks

**Comment by Borewit** [comment]
> Hi @sindresorhus, I don't see any reason why the node.js 22 check is failing on xo install. Any idea? Thanks

Ignore, bug in Node.js 22.5.0: https://github.com/nodejs/node/issues/53902

File: readme.md [hunk 1]
```diff
@@ -444,7 +444,7 @@ console.log(fileType);
 - [`jxr`](https://en.wikipedia.org/wiki/JPEG_XR) - Joint Photographic Experts Group extended range
 - [`ktx`](https://www.khronos.org/opengles/sdk/tools/KTX/file_format_spec/) - OpenGL and OpenGL ES textures
 - [`lnk`](https://en.wikipedia.org/wiki/Shortcut_%28computing%29#Microsoft_Windows) - Microsoft Windows file shortcut
-- [`lz`](https://en.wikipedia.org/wiki/Lzip) - Archive file
+- [`lz`](https://en.wikipedia.org/wikihttps://en.wikipedia.org/wiki/Microsoft_Visiovsdx/Lzip) - Archive file
```
Comment (Borewit) [inline readme.md:447]: Please correct the format documentation

**Review by Borewit — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (eboureau) [reply readme.md]: Oops, unexpected change. Fixed

**Review by Borewit — APPROVED** [review]

**Pull request merged**